src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -424,12 +514,12 @@ def _fetch_data(
                     MetricKey.USER,
                     metric_id,
                     [Function("uniq", [Column("value")], "value")],
+                    limit_state,
                 )
             )
             metric_to_output_field[(MetricKey.USER, "value")] = _UserField()
 
-    duration_fields = [field for field in query.raw_fields if "session.duration" in field]
-    if duration_fields:
+    if "session.duration" in raw_field:
```

Comment:
Is this supposed to be `==`?

Reply:
This was actually correct, but it took me a while to see it myself so I changed it into something more explicit.